applications/asset_tracker/src/main.c

```diff
@@ -988,8 +988,8 @@ static void modem_rsrp_handler(char rsrp_value)
 	 * Checking CONFIG_HOLD_TIME_RSRP gives the compiler a shortcut.
 	 */
 	if (CONFIG_HOLD_TIME_RSRP == 0 ||
-	    k_delayed_work_remaining_get(&rsrp_work) == 0) {
-		k_delayed_work_submit_to_queue(&application_work_q, &rsrp_work,
+	    k_ticks_to_ms_ceil32(k_work_delayable_remaining_get(&rsrp_work)) == 0) {
```

Comment:
This I believe need to be synchronized by other mechanisms

Reply:
Should use `k_work_schedule_for_queue` if we don't want to restart a running timer